src/gui/symbology/qgssymbollayerwidget.cpp

```diff
@@ -2839,6 +2851,15 @@ void QgsSVGFillSymbolLayerWidget::setFile( const QString &name )
   emit changed();
 }
 
+void QgsSVGFillSymbolLayerWidget::setSvgParameters( const QMap<QString, QgsProperty> &parameters )
+{
+  // TODO mLayer->setParameters(parameters);
```

Comment:
Still TODO?

Reply:
I will address this in a following PR to enable the params for SVG fills.